src/openrct2/management/Research.cpp

```diff
@@ -597,6 +597,12 @@ bool scenery_group_is_invented(int32_t sgIndex)
         return false;
     }
 
+    // All scenery is temporarily invented when in the scenario editor
+    if (gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR)
```

Comment:
```suggestion
    if (gScreenFlags & SCREEN_FLAGS_EDITOR)
```

Scenery is also available in the Track Designer.